tests/rolling_deploy_test.py

```diff
@@ -269,6 +273,19 @@ def test_double_autoscale_instance_count(self):
   def test_decrease_autoscale_instance_count(self):
     self.assertEqual(self.rolling_deploy.decrease_autoscale_instance_count(4), 2)
 
+  def test_cloudwatch_filter_function_with_no_alarm(self):
+    bad_alarm = MetricAlarm(name="xyz_CloudWatchAlarm", metric="CPUUtilization", namespace="AWS/EC2")
+    cloud_watch_alarms = filter(lambda alarm : self.rolling_deploy.project in alarm.name and self.rolling_deploy.env in alarm.name, [ bad_alarm])
+    print("alarms size {}".format(len(cloud_watch_alarms)))
```

Comment:
No need to print here or in the method below.

And in python 2.6, you need to add the number from the index into the {}.

So you should do: print("alarms size {0}".format(len(cloud_watch_alarms)))